docs/usage/self-hosted-configuration.md

```diff
@@ -553,6 +560,10 @@ You can choose from the following behaviors for the `dryRun` config option:
 
 Information provided mainly in debug log level.
 
+## eagerGlobalExtends
+
+Resolve and merge `globalExtends` presets before other global config, instead of after.
```

Comment:
```suggestion
By default, Renovate resolves and merges `globalExtends` presets _after_ other global config.

But if you set `eagerGlobalExtends` to ..., then Renovate will resolve and merge `globalExtends` presets _before_ the global config.
```

Before applying my suggestion, please:

- [ ] Double check if the sentences have the correct logic!
- [ ] Replace the `...` bit with the correct instruction